source/functions/initializeWecSim.m

```diff
@@ -463,7 +463,7 @@
 warning('off','MATLAB:printf:BadEscapeSequenceInFormat');
 warning('off','Simulink:blocks:DivideByZero');
 warning('off','sm:sli:setup:compile:SteadyStateStartNotSupported')
-set_param(0, 'ErrorIfLoadNewModel', 'off')
+
 % Load parameters to Simulink model
```

Comment:
My only concern about removing it here, is that not everyone uses the `addWecSimSource` script each time they open MATLAB. I think we should keep it in both places. I'm going to add it back here again too.